### PR TITLE
Moving plist info to NSUserDefaults

### PR DESCRIPTION
**Pull request by jwallra**
Plist didn't work on real device so moving info to `NSUserDefaults`

### CODE REVIEW

File: AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.m [hunk 1]
```diff
@@ -64,4 +68,8 @@ + (void) setWrapperSdk:(MSWrapperSdk *)sdk {
     [MSAppCenter setWrapperSdk:sdk];
 }
 
++ (void) setStartAutomatically:(BOOL)startedAutomatically {
```
Comment (dhei) [inline AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.m]: nit - What is the reason to name it `(BOOL)startedAutomatically`? I prefer `(BOOL)shouldStartAutomatically` or `(BOOL)startAutomatically`.

Comment (guperrot) [reply AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.m]: name conflict with the static variable

**Comment by guperrot** [comment]
There is an app freeze when overriding with appsecret only, don't merge yet.

File: AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.m (hunk 1 above)
Comment (guperrot) [reply AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.m]: On Android we have the getter `isStartedAutomatically`, removing the `ed` is odd.
should: then we would have a setter with `setShouldStartAutomatically` which would be odd.

On Android is just the couple is/setStartedAutomatically like every boolean "property".

File: AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.h [hunk 2]
```diff
@@ -14,4 +14,6 @@
 
 + (void) setWrapperSdk:(MSWrapperSdk *)sdk;
 
++ (void) setStartAutomatically:(BOOL)startedAutomatically;
```
[the remark this answers is not in the document]
Comment (guperrot) [reply AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.h]: It does not solve the conflict has a static property needs a setter.

File: AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.m (hunk 1 above)
Comment (TroubleMakerBen) [reply AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.m]: Just adding this for completeness:

If we define the static property like this

`@property (class, nonatomic, assign) BOOL isStartedAutomatically;`, provide an ivar `static BOOL _isStartedAutomatically = NO`;
and we define getter and setter like this

```
+ (BOOL)isStartedAutomatically {
  return _isStartedAutomatically;
}

+ (void)setIdentifier:(BOOL)isStartedAutomatically {
  _isStartedAutomatically = isStartedAutomatically;
}

``` there should not be any naming conflicts.

**Pull request merged**